src/main/java/logIn/interface_adapters/LogInPresenter.java

```diff
@@ -0,0 +1,27 @@
+package logIn.interface_adapters;
+
+import logIn.use_case.LogInOutputBoundary;
+import logIn.use_case.LogInResponseModel;
+
+public class LogInPresenter implements LogInOutputBoundary {
+    private final LogInViewModel viewModel;
+
+
+    public LogInPresenter(LogInViewModel viewModel){
+        this.viewModel = viewModel;
+
+    }
+
+
+    @Override
+    public void present(LogInResponseModel responseModel) {
+        if (responseModel.getLogInSuccess()){
```

Comment:
I realize that I have similar code in my file, but I think this is redundant. could just pass it without if/else